builtin/json.mbt

```diff
@@ -47,42 +47,8 @@ pub fn Double::to_json(self : Double) -> Json {
   Number(self)
 }
 
-fn escape_json_string(str : String) -> String {
-  fn to_hex_digit(i : Int) -> Char {
-    if i < 10 {
-      Char::from_int('0'.to_int() + i)
-    } else {
-      Char::from_int('a'.to_int() + (i - 10))
-    }
-  }
-
-  let buf = Buffer::new(size_hint=str.length())
-  for i = 0; i < str.length(); i = i + 1 {
-    let c = str[i]
-    match c {
-      '\n' => buf.write_string("\\n")
-      '\r' => buf.write_string("\\r")
-      '\b' => buf.write_string("\\b")
-      '\t' => buf.write_string("\\t")
-      _ => {
-        let code = c.to_int()
-        if code == 0x0C {
-          buf.write_string("\\f")
-        } else if code < 0x20 {
-          buf.write_string("\\u00")
-          buf.write_char(to_hex_digit(code / 16))
-          buf.write_char(to_hex_digit(code % 16))
-        } else {
-          buf.write_char(c)
-        }
-      }
-    }
-  }
-  buf.to_string()
-}
-
 pub fn String::to_json(self : String) -> Json {
-  String(escape_json_string(self))
+  String(self.escape())
```

Comment:
cc @hackwaly I remember this is not the same?

Reply:
Yes, Json escape is standardized, and our String::escape isn't. I suggest that keep the original version.

Reply:
I'm not sure if the original version follow the standard. Maybe we should add more tests here.